### DISCUSSION TITLE
p-galleria: Should have appendTo when fullScreen is true

### DISCUSSION THREAD

**Discussion by JavierExeni** [primefaces/community#1395] [PrimeNG]
**Describe the feature you would like to see added**
I want to be able to append the gallery to the body element, or some other outer element. Currently I have an issue with the stacking context.

**Is your feature request related to a problem?**
When we define the gallery as an inner component it can't be visualized on full screen on top of all other components.

**Describe the solution you'd like**
**_appendTo_** field would solve the issue

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback, we'll wait for community upvotes to implement this as a feature.

**Comment by MGPurnell** [comment]
I need this as well - would be very helpful in Version 19.
